radis/lbl/factory.py

```diff
@@ -648,7 +659,12 @@ def __init__(
     # XXX =====================================================================
 
     def eq_spectrum(
-        self, Tgas, mole_fraction=None, path_length=None, pressure=None, name=None
+        self,
+        Tgas,
+        mole_fraction=None,
+        path_length=None,
```

Comment:
you'll want `diluent` here too   (and in non_eq_spectrum() )  too

Reply:
I've added diluent in eq_spectrum() and non_eq_sprectrum() definition